README.md

```diff
@@ -1 +1,123 @@
-Initial
+# terraform-scalr-workspace
+Module to create and manage one or multiple workspaces.
+
+
+## Using the module
+### Compatibility
+- [Terraform](https://www.terraform.io/downloads.html) >= 0.13.x
+
+### Requirements
+
+```
+terraform {
+  required_version = ">=0.13"
+  required_providers {
+    scalr = {
+      source  = "registry.scalr.io/scalr/scalr"
+      version = "1.0.0-rc28"
+
+    }
+  }
+}
+
+provider "scalr" {
+  hostname = SCALR HOSTNAME
+  token    = SCALR TOKEN
+}
+```
+
+### Inputs
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| environment_id | Environment which workspace will be created in. | `string` | n/a | Yes |
+| name | Workspace name. | `string` | n/a | Yes |
+| auto_apply | Auto apply on successful plan. | `bool` | `false` | No |
+| operations | Set to configure workspace remote execution in CLI-driven mode. | `bool` | `true` | No |
+| terraform_version | Terraform version for workspace | `string` | `0.13.0` | No |
+| working_directory | A relative path that Terraform will be run in. | `string` | `null` | No |
+| run_operation_timeout | The number of minutes run operation can be executed before termination. | `integer` | `0` | No |
```

Comment:
```suggestion
| run_operation_timeout | The number of minutes run operation can be executed before termination. | `integer` | `120` | No |
```
The default timeout is 120 minutes. The bare minimum is 10 and if the TF resource will 0 zero it might fail with an error. Not sure, but need to double check it

Reply:
Thanks, I set the value based on the documentation - https://docs.scalr.com/en/latest/scalr-terraform-provider/resources/scalr_workspace.html. I can update it to 120 or leave it blank so that it picks up what the backend default is set to(probably 120).